packages/bot-web-ui/src/components/dashboard/tutorial-tab/faq-content.tsx

```diff
@@ -41,13 +41,26 @@ const FAQContent = observer(({ faq_list, hide_header = false }: TFAQContent) =>
     const timer_id = React.useRef<ReturnType<typeof setTimeout> | null>(null);
 
     const handleAccordionClick = () => {
-        // Scroll to the top of the open accordion item.
-        // Need timer to first close the accordion item.
+        // Scroll to the top of the open accordion item. Need timer to first close the accordion item then scroll the new item to top.
         timer_id.current = setTimeout(() => {
-            const openAccordionItem = document?.querySelector('.dc-accordion__item--open');
-            openAccordionItem?.scrollIntoView({ behavior: 'smooth', block: 'start' });
+            const faqWrapper = document?.querySelector('.faq__wrapper');
+            const openAccordionItem: HTMLElement | null = document?.querySelector('.dc-accordion__item--open');
+            const previousSibling = openAccordionItem?.previousElementSibling as HTMLElement;
```

Comment:
can we check the documentation if we are using `camelCase` for DOM entities?

Reply:
I am unaware of this. Let me check. It could be snake_case.

Reply:
Done ✅